consensus/src/dag/anchor_election/leader_reputation_adapter.rs

```diff
@@ -133,7 +133,11 @@ impl LeaderReputationAdapter {
         &self,
         round: u64,
     ) -> (f64, Option<&aptos_config::config::ChainHealthBackoffValues>) {
-        let voting_power_ratio = self.reputation.get_voting_power_participation_ratio(round);
+        let mut voting_power_ratio = self.reputation.get_voting_power_participation_ratio(round);
+        // TODO: fix this once leader reputation is fixed
+        if voting_power_ratio < 0.67 {
+            voting_power_ratio = 1.0;
+        }
```

Comment:
How can this even be below 0.67?
And why do we set it to 1?
Temporary hack?

Reply:
Yes, definitely temporary. When we start epoch, this can be less than 0.67 because we don't yet read the previous epoch events.